testing/tests/test_e2e/test_deferred.py

```diff
@@ -16,13 +20,13 @@
 @pytest.fixture(scope='function')
 def mycharm():
     class MyCharm(ops.CharmBase):
-        META: dict[str, typing.Any] = {
+        META: typing.ClassVar[dict[str, typing.Any]] = {
```

Comment:
This is the 'correct' type annotation, but `Mapping` would feel cleaner to me if we don't mutate `META` in tests.
```suggestion
        META: Mapping[str, typing.Any] = {
```